sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -1,17 +1,11 @@
 # Release History
 
-## 1.11.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.11.0-beta.1 (2023-09-06)
```

Comment:
This should be the 13th, no?   Are you intending to release early?

Reply:
No, I just let the release script do its thing. I'll adjust.